.cirrus.yml

```diff
@@ -167,6 +167,17 @@ centos9_build_task:
     script:
         - cargo build
 
+aws_credentials: ENCRYPTED[3fab904a98355f84b0bac084f8a50428ff8a27dd2b6a6c42fca77df89010e620a1da3cd246a50b1074e6787c42818080]
+
+#image: ami-0d15f8a570d61e229
+#image: ami-01925eb0821988986
+fedora36_aarch64_task:
+  image: ami-0b8b7e5c3c2df76c7
```

Comment:
This line needs to be nested inside the `ec2_instance` map down below :grin:

Reply:
See example here: https://cirrus-ci.org/guide/supported-computing-services/#ec2

Reply:
done